x-pack/plugins/security_solution/server/endpoint/routes/actions/response_actions.test.ts

```diff
@@ -0,0 +1,555 @@
+/*
```

Comment:
This is 90% copied from https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/server/endpoint/routes/actions/isolation.test.ts. Will point out differences in comments.